src/core/scene/a-scene.js

```diff
@@ -169,7 +169,7 @@ var AScene = module.exports = registerElement('a-scene', {
         if (!camera || !canvas) { return; }
 
         // Update canvas.
-        if (!isMobile) {
+        if (!isMobile && !canvas.provided) {
```

Comment:
yes, my personal use case is that i'd like to be able to dedicate some subset of the page to my aframe scene and have other html surrounding it. I would be assuming control over how much of the page my canvas occupies.